src/main/java/beam/agentsim/events/handling/BeamEventsWriterCSV.java

```diff
@@ -89,7 +89,9 @@ protected void writeEvent(Event event) {
                 String str = row[i];
                 if (str != null) {
                     if (str.contains(",")) {
-                        this.out.append("\"" + str + "\"");
+                        this.out.append('"');
```

Comment:
![image](https://user-images.githubusercontent.com/5107562/44106085-d65c307c-a01d-11e8-878c-24ec2f2d00d4.png)

Reply:
Comments: Good job here!

Recommended suggestions:
- Use fluent interface. It will become a little bit faster because will avoid some indirections and some garbage

Suggestion

```Java
out.append('"')
    .append(str)
    .append('"');
```